src/stylesheets/components/_nav.scss

```diff
@@ -59,3 +65,45 @@
     }
   }
 }
+
+.pillbar-fixed {
+  background-color: $florence-red;
+  box-shadow: inset 0 -16px 0 rgba(0, 0, 0, 0.1);
+  margin: auto;
+  position: fixed;
+  top: 0;
+
+  li {
+    box-shadow: inset 0 -16px 0 rgba(0, 0, 0, 0.1);
+
+    &:first-child, &:last-child {
+      border-radius: 0;
+    }
+  }
+}
+
+.pillbar::before {
+  background-color: $florence-red;
+  border-radius: 1px;
+  box-shadow: inset 0 -16px 0 rgba(0, 0, 0, 0.1);
+  content: '';
+  height: 35px;
+  position: absolute;
+  top: 0;
+  transform: scaleX(1);
```

Comment:
why do you need this transform? looks like it should be transitioning but i don't see where it gets modified.